### ISSUE TITLE
Don't enable the Sessions schedule Display.

### ISSUE THREAD

**Issue by esod** [NYC-Camp/website#249]
@all. Please don't enable the The Session Schedule display in http://www.nyccamp.org/admin/structure/views/view/schedule_sessions/edit/page_3 or in http://www.nyccamp.org/admin/structure/views/view/nyccamp_schedule/edit/page_3. Those displays don't work and they each place a menu item in the main menu with default COD text on them. I just found them enabled on live where they disrupted the theme by pushing the main menu onto a line below the logo.

Really that's why I built the Personal Schedule module.

**Comment by esod** [comment]
I add the Add to my sessions/Remove from my sessions flag to the display on http://www.nyccamp.org/program/sessions-schedule. The placement of the flag isn't perfect but I'm working on it and the UX works with the Personal Schedule module. 

Btw, Annie's link in the post ( https://groups.drupal.org/node/417453 ) isn't working. 

is:
https://www.nyccamp.org/program/sessions-schedule
should be:
http://www.nyccamp.org/program/sessions-schedule

**Comment by ForestMars** [comment]
Team, we have to tighten up here and get on the same page. 

We can't have some people pushing the schedule live while others are saying it's not ready. At the very least we need better communication, even if we don't have agreement about whether it's ready to go live, or whether we can't wait any longer. 

We need to pick a canonical channel, either Skype or IRC, and stick with it. And/or continue to use Trello for the high level milestones & tasks, such as publishing the my sessions schedule (although we didn't include that feature in the official announcement, right?)

OTOH, why do we have views on Live if they are not ready to be enabled?

**Comment by esod** [comment]
The schedule flag is finished. I'll get the placement perfect later tonight. It all works. My question is why do we have duplicate views? Why were they both enabled on live and not checked for their menu links? I am also trying to get the existing My Profile links to go to the correct tab. Right now they default to /user. That's not simple.

I think our priority is to reach Annie and ask her to fix her link.

The personal schedule module should be included in the official announcement. It's okay if I have do a little bit of my own marketing for it so it doesn't get lost in the marketplace of ideas.

**Comment by ForestMars** [comment]
We can't "fix" the link, it's already gone out in an email campaign. 

And while it's okay for anyone to do their own "marketing" to promote some aspect of the camp (in fact it's not just ok, it's awesome) the goal should be to bubble up key pieces such as "Add to my schedule" into the official email announcements. In fact that one in particular is a near perfect example of a key message that should be communicated to all attendees.

**Comment by esod** [comment]
Thank you @ForestMars. :)

And now back to the grind ...

**Comment by willykaram** [comment]
Guys sorry for reply late here. We aren't using the view that Eric mentions above and it should be disabled/made in accessible on the site. This is the old view from last year. We'll need to think about how we want to handle the multi-year functionality, but that can wait until after the camp. FWIW, I don't think that anyone "enabled" this display recently, I think it's largely been there since we started work on this year's site. In order to close this issue, I suggest that we disable the above view as we are not using any of it anymore for 2014.

Note that this is the view for the schedule that was published this year:
- Back-End: http://www.nyccamp.org/admin/structure/views/view/schedule_sessions_keynotes/edit/page_1
- Front-End: 
  http://www.nyccamp.org/program/sessions-schedule 

Other individuals views are being used for summits and other scheduling needs (mostly blocks and EVAs); and we'll hopefully have pretty much all of that finalized today.  Note that some of the documentation relating to the new scheduling implementation is available here:
- https://github.com/NYC-Camp/website/wiki/NYC-Camp-Conference-Scheduling

**Comment by willykaram** [comment]
BTW, sorry for not replying sooner, but I just saw this today/now.

**Comment by esod** [comment]
Not one of the displays in this view:

http://www.nyccamp.org/admin/structure/views/view/schedule_sessions

gets any data. I'm disabling the view.

**Comment by willykaram** [comment]
Thanks, agreed

**Comment by esod** [comment]
Likewise:

http://www.nyccamp.org/admin/structure/views/view/nyccamp_schedule

which is now disabled.

I am certain I deleted the displays that messed with the Personal Schedule concept and execution and added two nonsensical menu items to the main menu as I described at the top of this issue. But I'm sure there was no malice intended when this event occurred.

**Comment by ForestMars** [comment]
Just to clarify, this view _is_ used to generate the schedule for last year. 

Since the data model was completely retooled this year, this view is not used to generate this year's schedule, but _is_ responsible to displaying last year's schedule. 

If there's an issue with it that we are not resourced to fix before the camp, I'm fine with temporarily disabling it, but as Willy alludes to, we need to think about how we're handling multi-year functionality, and this is currently the way it was implemented for 2013.

**Comment by esod** [comment]
Good point. There are three data models in our system that handle scheduling. I suggest we meet after the camp, decide on one data model, port the old nodes to that data model and delete the other two data models. Yes, delete them because it's very cluttered and confusing now. 

Time session slot(s), Time session(s), and whatever the new one can be recognized by; ..._evnt or something. Which I don't understand because it was never explained and the whole data model just appeared on the site two weekends ago. But that's an old saw. 

I would like to understand the _evnt data model because the History of Drupal session has something the matter with data that's been input into it and that session won't sort properly in My Sessions.

**Comment by esod** [comment]
... and I can't fix it because I don't know how its Time slot (and location, I think) have been attached to it.

**Comment by esod** [comment]
I'm thinking about our memory footprint by disabling both views and both views seem so similar to me.  @ForestMars which view would you like re-enabled for last year's schedule?

**Comment by ForestMars** [comment]
I don't know offhand that last year's view is actually working on the site? ie. that when enabled it provides a display of last year's sessions. 

Also, we have added the path /2014 for all events (and sessions) posted for this year. I'm pretty sure that's not set up to work for last year. 

In any case, I'm pretty sure the correct view which is used to generate all the schedule views for last year is http://www.nyccamp.org/admin/structure/views/view/nyccamp_schedule

As you pointed out, it's not returning any results, most likely bc someone messed with a field or entity ref. Are the relationships on this view even working?
